src/GitVersionCore/GitVersionException.cs

```diff
@@ -0,0 +1,42 @@
+﻿#region License
+
+// --------------------------------------------------
+// Copyright © PayEx. All Rights Reserved.
+// 
+// This software is proprietary information of PayEx.
+// USE IS SUBJECT TO LICENSE TERMS.
+// --------------------------------------------------
+
+#endregion
+
+namespace GitVersion
+{
+    using System;
+    using System.Runtime.Serialization;
+
+    [Serializable]
+    public class GitVersionException : ApplicationException
```

Comment:
Would you consider deriving from `GitToolsException` instead : `https://github.com/GitTools/GitTools.Core/blob/11fa325b9f7afa0f9d8abf12d260210f82a29e15/src/GitTools.Core/Exceptions/GitToolsException.cs` - It will make it a bit easier to get this building under netstandard1.3 where `ApplicationException` isn't avialable, and it will also make exceptions accross the tools more consistent which I think was the purpose behind GitToolsException.

Reply:
@dazinator: Sure. Done.